packages/patternfly-4/react-catalog-view-extension/package.json

```diff
@@ -43,17 +43,8 @@
     "@patternfly/patternfly": "2.43.0",
     "@patternfly/react-core": "^3.122.1",
     "@patternfly/react-styles": "^3.6.10",
-    "breakjs": "^1.0.0",
     "classnames": "^2.2.5",
-    "css-element-queries": "^1.0.1",
-    "patternfly": "^3.59.4",
-    "patternfly-react": "^2.39.9",
```

Comment:
We do need PF React for the catalog tiles.

Reply:
@rebeccaalpert we need pf3 for catalog tiles? I didn't see any imports for the package when searching the catview extension (minus a link in the text of one of the unit test cases

Reply:
Looks good to me.